xdsl/rewriter.py

```diff
@@ -141,6 +141,24 @@ def insert_block_before(block: Block | List[Block], target: Block):
         pos = region.get_block_index(target)
         region.insert_block(block_list, pos)
 
+    @staticmethod
+    def insert_op_after(op: Operation, new_op: Operation):
```

Comment:
AFAIU, you could pretty much insert ops everywhere and anywhere. This might be very dangerous. Notice that this does not concern our rewriter, as we don't actually call this, just pointing it out.